datastore/snippets/snippet_test.go

```diff
@@ -507,36 +507,43 @@ func SnippetIterator_Cursor() {
 	ctx := context.Background()
 	client, _ := datastore.NewClient(ctx, "my-proj")
 	defer client.Close()
-	cursorStr := ""
 	// [START datastore_cursor_paging]
-	const pageSize = 5
-	query := datastore.NewQuery("Tasks").Limit(pageSize)
-	if cursorStr != "" {
-		cursor, err := datastore.DecodeCursor(cursorStr)
-		if err != nil {
-			log.Fatalf("Bad cursor %q: %v", cursorStr, err)
+	for {
+		cursorStr := ""
+		const pageSize = 5
+		query := datastore.NewQuery("Tasks").Limit(pageSize)
+		if cursorStr != "" {
```

Comment:
I think at this point, `cursorStr` will always be `""`. Should `cursorStr` be initialized outside the loop?